src/commands/leaderboard.py

```
@@ -28,102 +28,370 @@
+        case((correct_attr.is_(True), 1), else_=0)
+    )
+
+    query = select(User, total_correct.label("total_correct")).join(
+        Pick, getattr(Pick, "user_id") == User.id
+    )
+
+    # Apply optional filters
+    if days is not None:
+        cutoff = datetime.now(timezone.utc) - timedelta(days=days)
+        if hasattr(Pick, "created_at"):
+            query = query.where(getattr(Pick, "created_at") >= cutoff)
```

Ruff (0.14.10)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)